Modules/Core/Common/src/itkPoolMultiThreader.cxx

```diff
@@ -41,16 +41,22 @@ namespace
 class ExceptionHandler
 {
 public:
+  ExceptionHandler() = default;
```

Comment:
If you really want to remove the explicit `ExceptionHandler(const TFunction &)` constructor, you might as well follow the Rule of Zero. (It is not necessary to declare the default-constructor anymore.)

Reply:
done.